tasks/task1437_doqa_cooking_question_generation.json

```diff
@@ -0,0 +1,5525 @@
+		"English"
+	],
+	"Categories": [
+		"Question Generation"
+	],
+	"Definition": "Given a paragraph about cooking, and a set of conversational question answers about the paragraph, generate a relevant question to the topic of the paragraph. The paragraph has a prefix 'CONTEXT:'. Each conversation question has a prefix Q: followed by answer prefix A: ",
+	"Positive Examples": [
+		{
+			"input": "CONTEXT: Milk won't help you - it's mostly water, and gluten develops from flour (more accurately, specific proteins in flour) and water.The way to reduce gluten development is to incorporate more fat into the batter. Lipids are hydrophobic and will prevent further hydration of the glutenin.Using a lower-protein flour will also help. If you're not already using cake flour, the reason it's called cake flour is because of the lower protein content.That being said, have you actually tried leaving the batter coarse? Just because the batter is lumpy does not mean that the cake will have big lumps. The entire mixture is wet, so unless you leave huge lumps of dry flour in the batter, you won't end up with a lumpy cake. There's a difference between don't overmix and don't mix - you're supposed to mix enough to incorporate, just don't try homogenize it. Q: How can I avoid lumps in the cake? A: have you actually tried leaving the batter coarse? Just because the batter is lumpy does not mean that the cake will have big lumps Q: No, I haven't. I'll try that next. Is it safe to add more milk into the batter? A: Milk won't help you - it's mostly water, and gluten develops from flour (more accurately, specific proteins in flour) and water.",
+			"output": "What can I add to make it more softer and avoid hardening?",
+			"explanation": "This conversation is about the process of preparing cakes and how lumps can be avoided in it. The question asked is relevant as it is on the same topic and it asks how to avoid hardening of cake. This is also a followup question of the last conversational question and answer about adding more milk into the cake batter."
+		},
+		{
+			"input": "CONTEXT: It not huge, it's just the difference from freezer to room temperature you are worrying aboutE.g. -20\u00b0C to 20\u00b0C, is A 40\u00b0C shift. The shift was going to be 20\u00b0C to 100\u00b0C anyway. There is no physical reasons why this would be anymore stressfulFrom room temperature you are raising it 80\u00b0C, from frozen you are raising it 120\u00b0C. Not a problem in the normal temperature range for glassThe freezing temperature of water (most common item in food) has no relation to the freezing temperature of glass etc Pyrex and other glasses can be damaged if one part of them is instantly heated or cooled by a 100\u00b0C or so Q: Is it safe to microwave Pyrex containers immediately after removing them from the freezer and removing the plastic lid? A: Pyrex and other glasses can be damaged if one part of them is instantly heated or cooled by a 100\u00c2\u00b0C or so",
```

Comment:
- Make sure that your json is human readable (use proper indentation; e.g., in Python: json.dumps(your_json_string, indent=4, ensure_ascii=False))
- Please use `<sep>` token for separating context and conversations.